editor/filesystem_dock.cpp

```diff
@@ -1296,6 +1296,13 @@ void FileSystemDock::_fs_changed() {
 		_update_file_list(true);
 	}
 
+	if (!select_after_scan.is_empty()) {
+		navigate_to_path(select_after_scan);
+		select_after_scan = String();
+		import_dock_needs_update = true;
+		_update_import_dock();
```

Comment:
Do you know why we call it deferred everywhere else?

Reply:
Probably to avoid multiple updates in a single frame. There is no such risk here.